docs/core/tools/dotnet-test.md

```diff
@@ -79,6 +85,10 @@ Test projects specify the test runner using an ordinary `<PackageReference>` ele
 
   Doesn't build the test project before running it. It also implicitly sets the - `--no-restore` flag.
 
+- **`--nologo`**
+
+  Run tests without displaying the Microsoft TestPlatform banner. Available since .NET Core 3.0 SDK.
```

Comment:
I think the other articles you did that had an "Available since ... " clause used a different format. I think they were bold?